Sources/Components/HorizontalSlide/HorizontalSlideTransition.swift

```diff
@@ -5,10 +5,50 @@ import UIKit
 }
 
 public class HorizontalSlideTransition: NSObject, UIViewControllerTransitioningDelegate {
+
+    public struct ContainerSize {
+
+        let portrait: CGFloat
+        let landscape: CGFloat
+
+        var percentage: CGFloat {
+            return UIDevice.current.orientation.isLandscape ? self.landscape : self.portrait
+        }
+
+        public init(portrait: CGFloat, landscape: CGFloat? = nil) {
+            self.portrait = portrait
+            self.landscape = landscape ?? portrait
+        }
+
+        public static var `default`: ContainerSize {
+            if UIDevice.isIPad() {
+                return ContainerSize(portrait: 0.60)
+            } else {
+                return ContainerSize(portrait: 0.85)
+            }
+        }
+    }
+
     @objc public weak var delegate: HorizontalSlideTransitionDelegate?
 
+    private let containerSize: ContainerSize
+
+    // MARK: - Init
+
+    public init(size containerSize: ContainerSize) {
```

Comment:
I would keep only `size` or `containerSize` here, to me it doesn't really matter which one it is ;)